### PR TITLE
codeowners: add owner for tests of the nrf_cloud library

### PR DESCRIPTION
**Pull request by tony-le-24**
Add my GitHub handle as owner for test cases that are in /tests/subsys/net/lib/nrf_cloud/ 
This folder is not added yet since PR#8418, which includes it, is opening

See NCSDK-16128

Signed-off by: Tony Le <tony.le@nordicsemi.no>

### CODE REVIEW

**Review by plskeggs — APPROVED** [review]
The errors on lines 120 and 121 seem to be due to a recently departed employee, maybe.

Your change seems fine.  But because of the unrelated errors, I wonder if this will be mergeable?

**Comment by gopiotr** [comment]
I made PR which will solve current problem with `ppryga` code owner nick name, here: https://github.com/nrfconnect/sdk-nrf/pull/8436

But even if you rebase this PR you will still have a problem with merge it due to the fact that you cannot put new register into CODEOWNER file if there still do not exist path like `/tests/subsys/net/lib/nrf_cloud/` in current tree.

So, I propose you two solutions:
1. Close this PR, and add commit which modify appropriately CODEOWNERS in this PR: https://github.com/nrfconnect/sdk-nrf/pull/8418. This is a standard way how we modify CODEOWNERS file (simultaneously with adding new tests in the same PR). For example you can observe how it worked here: https://github.com/nrfconnect/sdk-nrf/pull/7927/files (partition_manager tests were added and changes in CODEOWNERS file were introduced too).

or

2. Wait with merge this PR, until this one will be merged firstly: https://github.com/nrfconnect/sdk-nrf/pull/8418 and then rebase this PR to the head to make it possible to "see" `/tests/subsys/net/lib/nrf_cloud/` path in tree.

Please make a decision and tell me which path you will follow ;)

**Comment by tony-le-24** [comment]
@gopiotr I will choose the second route, wait for my other PR got merged and proceed with this one. This is for now. In the future, if a case like this happens, I will follow the first option. Thanks.

**Comment by tony-le-24** [comment]
I decided to close this one and add the changes to the PR#8418

**Pull request closed without merging**
